lmdeploy/pytorch/config.py

```diff
@@ -109,14 +110,16 @@ def __build_chatglm():
             bos_token_id = hf_config.bos_token_id
             if bos_token_id is None:
                 bos_token_id = hf_config.pad_token_id
+            init_kwargs = dict(empty_init=False)
```

Comment:
why need to add `empty_init=False`?

Reply:
init_empty_weight in accelerate can not load chatglm to `meta` device with `empty_init=True`, which leads to tp adapter load failed.

Reply:
How to test this model?
The following code failed
```python
pipe = pipeline('/workspace/models-140/chatglm3-6b/', 
                backend_config=PytorchEngineConfig(
                    model_name='base'
                ))
```

Reply:
> The following code failed

template failed, fixed in #1168

Reply:
tp=1 pass
tp=2 failed

Reply:
May update FAQs in pipeline.md
The current phenomenon is 
```
raise RuntimeError(f'Rank[{idx}] failed.')
```

Reply:
Done